### PR TITLE
feat: Implementation Datax doriswriter plugin

### PR DESCRIPTION
**Pull request by Kyofin**
## Proposed changes

Implementation datax doriswriter plugin. 
To compile this plugin , you need to download [Alibaba/Datax](https://github.com/alibaba/DataX) source and put plugin to that maven project.

## Types of changes

- [x] New feature (non-breaking change which adds functionality)

## Checklist

- [ ] I have created an issue on (Fix #ISSUE) and described the bug/feature there in detail
- [ ] Compiling and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] If these changes need document changes, I have updated the document

## Further comments
Should I write a doc about how to install this  plugin in Datax?

### CODE REVIEW

**Comment by morningman** [comment]
Hi @huzk8 ,this PR #6111 has been merged, please move the `doriswriter` dir to the `extension/DataX/doriswriter`.

File: extension/datax-doriswriter/src/main/java/com/alibaba/datax/plugin/writer/doriswriter/DorisWriterEmitter.java [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+	/**
+	 * execute doris stream load
+	 */
+	public void doStreamLoad(final DorisFlushBatch flushData) throws IOException {
+		final String host = this.getAvailableHost();
+		if (null == host) {
+			throw new IOException("None of the host in `beLoadUrl` could be connected.");
+		}
+		final String loadUrl = host + "/api/" + this.keys.getDatabase() + "/" + this.keys.getTable() + "/_stream_load";
+		LOG.debug(String.format("Start to join batch data: rows[%d] bytes[%d] label[%s].", flushData.getRows().size(), flushData.getBytes(), flushData.getLabel()));
```
Comment (hf200012) [inline extension/datax-doriswriter/src/main/java/com/alibaba/datax/plugin/writer/doriswriter/DorisWriterEmitter.java]: It is recommended to use info here to facilitate troubleshooting
debug -> info

**Review by hf200012 — COMMENTED** [review]
OK

**Review by morningman — DISMISSED** [review]
LGTM

**Review by morningman — APPROVED** [review]
LGTM

**Pull request merged**